app.rb

```diff
@@ -12,6 +15,10 @@
   overRustleLogsParser.get_chat.to_json
 end
 
+get '/emotes' do
+  emotes

```

Comment:
I think this line can just be `open('https://cdn.destiny.gg/2.13.0/emotes/emotes.json')`

Reply:
This way is simpler, but how it is right now it only gets the .json from the cdn on server launch and not on every /emotes request. Maybe worth keeping it?

Heroku dynos apparently get restarted every 24 hours anyway, so stuff should still get updated.

Reply:
Ah gotcha ok lets keep it the way it is